packages/grpc-js/src/client.ts

```diff
@@ -198,9 +198,9 @@ export class Client {
     options: CallOptions;
     callback: UnaryCallback<ResponseType>;
   } {
-    if (arg1 instanceof Function) {
+    if (Object.prototype.toString.call(arg1) === '[object Function]') {
```

Comment:
Can you extract this condition into an `isFunction` check?

Reply:
certainly, in this file itself, outside the class?